enterprise/backend/src/metabase_enterprise/audit_app/audit.clj

```diff
@@ -223,9 +224,59 @@
     (when-let [audit-db (t2/select-one :model/Database :is_audit true)]
       (adjust-audit-db-to-host! audit-db))))
 
+(def ^:private audit-db-entity-id
+  "Hard-coded `:entity_id` for the audit DB. Used to compute any missing `:entity_id`s for existing audit DBs."
+  "audit__rP75CiURKZ-0pq")
+
+(defn- entity-id-for-table [table]
+  (-> [audit-db-entity-id
+       ;; The hard-coded entity_ids saved in the serdes export used a schema of "public", so that's now hard-coded.
+       ;; The schema (and spelling) used for the AppDB tables varies by engine, so it should not influence the idents.
+       "public"
+       ;; We use inconsistent upper and lower case table and field names for audit across AppDB engines; this uses
+       ;; lower case everywhere to make the entity IDs effectively hard-coded.
+       (u/lower-case-en (:name table))]
```

Comment:
I don't quite follow: if the spelling for app-db tables varies by engine, and we use the name of the table in the hash, then wouldn't that imply that the name will influence the eid? 

I see it doesn't in the tests, so maybe I'm missing something from the comment.

Reply:
Ah, I should adjust the phrasing of these comments. The intent is that even though normally there *are* different schemas, spellings etc. we're deliberately papering over them here so that the `entity_id`s are effectively hard-coded, and identical across engines.